docs/references/protocol/transactions/types/ledgerstatefix.md

```diff
@@ -0,0 +1,68 @@
+---
+seo:
+    description: Repair corruptions to the XRP ledger.
+labels:
+  - Utilities
+  - Troubleshooting
+---
```

Comment:
This won't do anything _yet_, but I hope to extend Redocly's sidebar so that we can use this frontmatter field to make the "not enabled" flask icon appear in the sidebar again:

```suggestion
---
seo:
    description: Repair corruptions to the XRP ledger.
labels:
  - Utilities
  - Troubleshooting
status: not_enabled
---
```